webapp/channels/src/components/profile_popover/profile_popover_controller.tsx

```diff
@@ -127,6 +137,7 @@ export function ProfilePopoverController<TriggerComponentType = HTMLSpanElement>
                                 ref={refs.setFloating}
                                 style={{...floatingStyles, ...transitionStyles}}
                                 className={classNames('user-profile-popover', A11yClassNames.POPUP)}
+                                aria-label={props.username ? `${props.username}'s ${profilePopoverLabel}` : profilePopoverLabel}
```

Comment:
This needs to be localized, the whole sentence - `${props.username}'s ${profilePopoverLabel}` should go into the localized text. Not all language follow the structure of <proper noun>'s <something>. Additionally this will leave the `s` out of localized text.